docs/upgrade-to-v5.md

```diff
@@ -187,17 +187,23 @@ Model.findAll({
 - Sequelize now works with `tedious >= 6.0.0`. Old `dialectOptions` has to be updated to match their new format. Please refer to tedious [documentation](http://tediousjs.github.io/tedious/api-connection.html#function_newConnection). An example of new `dialectOptions` is given below
 
 ```json
-dialectOptions: {
-  authentication: {
-    domain: 'my-domain'
-  },
-  options: {
-    requestTimeout: 60000,
-    cryptoCredentialsDetails: {
-      ciphers: "RC4-MD5"
-    }
-  }
-}
+database: 'DB_NAME',
```

Comment:
Just want to keep the config minimum, `type`, `port` & `requestTimeout` already got defaults. No need to supply `username`, `password` or `host` as sequelize will supply them.

This documentation should only show how you can pass extra `authentication` and `options` for `tedious` no need to complicate it :)